_layouts/default.html

```diff
@@ -11,6 +11,8 @@
             });
         });
     </script>
+    <link rel="stylesheet" href="/assets/css/sig.css">
```

Comment:
We don't want this stylesheet on every page. so link it on individual sigs HTML page - I think events uses a `js` and `css` parameter, follow that